Gems/AtomLyIntegration/AtomFont/Code/Source/FFont.cpp

```diff
@@ -1435,23 +1434,17 @@ bool AZ::FFont::InitTexture()
 {
     using namespace AZ;
 
-    RHI::Format rhiImageFormat = RHI::Format::R8_UNORM;
-    int width = m_fontTexture->GetWidth();
-    int height = m_fontTexture->GetHeight();
-    uint8_t* fontImageData = m_fontTexture->GetBuffer();
-    uint32_t fontImageDataSize = RHI::GetFormatSize(rhiImageFormat) * width * height;
+    const RHI::Format rhiImageFormat = RHI::Format::R8_UNORM;
+    const int width = m_fontTexture->GetWidth();
+    const int height = m_fontTexture->GetHeight();
+    const Name imageName(m_name.c_str());
 
-    Data::Instance<RPI::StreamingImagePool> streamingImagePool = RPI::ImageSystemInterface::Get()->GetSystemStreamingPool();
-    m_fontStreamingImage = RPI::StreamingImage::CreateFromCpuData(
-        *streamingImagePool.get(),
-        RHI::ImageDimension::Image2D,
-        RHI::Size(width, height, 1),
-        rhiImageFormat,
-        fontImageData,
-        fontImageDataSize);
+    Data::Instance<RPI::AttachmentImagePool> imagePool = RPI::ImageSystemInterface::Get()->GetSystemAttachmentPool();
+    RHI::ImageDescriptor imageDescriptor = RHI::ImageDescriptor::Create2D(RHI::ImageBindFlags::ShaderReadWrite, width, height, rhiImageFormat);  // TODO check flag
```

Comment:
I set ShaderReadWrite, but I'm not sure at all. Please check. Thanks!

Reply:
ShaderReadWrite is fine for the use case. The image was actually has CopyWrite and ShaderRead. 
But the low level implementation added CopyWrite by default when init image in Vulkan. And DX12 is flexible with resource state. So ShaderReadWrite works fine. 
If you change to ShaderRead only then it might be an issue with some high level validation which I am not sure of.